salt/hg/config/hg.apache.conf.jinja

```diff
@@ -1,5 +1,13 @@
+#Configure rate limiting
+QS_ClientEventLimitCount 10 60 rate_limit
+QS_ClientEventPerSecLimit 1
+QS_ErrorResponseCode 429
+
 <VirtualHost _default_:9000>
-    ServerName hg.python.org
+    SetEnvIf Request_URI / rate_limit=1
+
+    ServerName hg.psf.io
```

Comment:
I don't think we should ship this change to production. Please refer the ServerName/ServerAlias parts.

Reply:
this makes total sense